### ISSUE TITLE
Make "Verify phone…" a submit button

### ISSUE THREAD

**Issue by tomalec** [woocommerce/google-listings-and-ads#1036]
## User story

As a merchant verifying phone number, especially one using the keyboard as input, I'd like to easily submit a verification code, by filling the code then pressing <kbd>enter</kbd>

## Technical

As we are not using native `<form>` and `<submit>`/`<button type="submit">` HTML elements, those actions does not work out of the box.

## Figma link
n/a

**Comment by puntope** [comment]
@tomalec 

I'm working on this issue.

Could you explain to me step by step in order to find that phone number input in the plugin admin page?

**Comment by puntope** [comment]
> Could you explain to me step by step in order to find that phone number input in the plugin admin page?

1. Setup Merchant Page 
2. Section 4 (Confirmation page)

![Screenshot 2021-11-17 at 13 11 26](https://user-images.githubusercontent.com/5908855/142198316-9add5412-b947-4816-8fae-5c6d02bdcf11.png)

**Comment by tomalec** [comment]
> Could you explain to me step by step in order to find that phone number input in the plugin admin page?

Sorry, that I didn't add detailed steps in advance. I created it in the times when the flows and URLs where this component was available were quite volatile. And I consider this a feature request for a single component, regardless of its placement.

Yep, that's one way to get this view:
#### Setup
1. Start the setup `/wp-admin/admin.php?page=wc-admin&path=%2Fgoogle%2Fsetup-mc`
2. Complete the setup through 4th step `/wp-admin/admin.php?page=wc-admin&path=%2Fgoogle%2Fsetup-mc&google-mc=connected`
3. Provide a real phone number
![go to phoneverification](https://user-images.githubusercontent.com/17435/142239820-d9c9354e-426a-40d2-a54d-f78be2826e30.gif)

#### Settings
The other is, once you're set up
1. Go to settings
2. Click "Edit" in phone number card, and go to [`/wp-admin/admin.php?page=wc-admin&subpath=%2Fedit-phone-number&path=%2Fgoogle%2Fsettings`](https://wp581wc580.test/wp-admin/admin.php?page=wc-admin&subpath=%2Fedit-phone-number&path=%2Fgoogle%2Fsettings)
